simglucose/analysis/report.py

```diff
@@ -243,12 +247,16 @@ def CVGA(BG_list, label=None):
     return zone_stats, fig, ax
 
 
-def report(df, save_path=None):
+def report(df, cgm_sensor=None, save_path=None):
     BG = df.unstack(level=0).BG
 
+    sample_rate = 20  # Use 20 as default sample rate in one hour
+    if cgm_sensor is not None:
+        sample_rate = int(60 / cgm_sensor.sample_time)
```

Comment:
Also echo my first comment, you can parameterize this 60 into a `window_length` parameter in the `risk_index_trace` function.